docs/content/en/search/_index.md

```diff
@@ -0,0 +1,5 @@
+---
+date: 2022-3-31

```

Comment:
Why do we need this file?

Reply:
Just the first commit is going to be merged, the second one was added so you can test the feature :D This is mentioned in the description of the PR :D